collect_events.py

```diff
@@ -1,24 +1,71 @@
 import json
+import logging
 import os
+import time
 
+import jwt
 import requests
 
-# Twosense API access token. Get it here:
-# https://twosense.readme.io/reference/authentication#2-obtain-an-access-token
-API_TOKEN = os.getenv("TWOSENSE_API_TOKEN")
+logging.basicConfig(
+    filename="app.log",

```

Comment:
should we add the app.log file to gitignore?

Reply:
It's already git ignored.